clients/imodels-client-management/src/operations/imodel/IModelOperations.ts

```diff
@@ -82,7 +82,16 @@ export class IModelOperations<TOptions extends OperationOptions> extends Operati
    */
   public async createEmpty(params: CreateEmptyIModelParams): Promise<IModel> {
     const createIModelBody = this.getCreateEmptyIModelRequestBody(params.iModelProperties);
-    const createdIModel = await this.sendIModelPostRequest(params.authorization, createIModelBody, params.headers);
+    let createdIModel = await this.sendIModelPostRequest(params.authorization, createIModelBody, params.headers);
+    
+    if (params.iModelProperties.creationMode) {
```

Comment:
I think it would be better to rely on createdIModel state. We should wait if iModels is "in progress" of initialization.

Reply:
Yes for sure, thanks. Updated.